### PR TITLE
Cleans up chat on exit, and other chat-related UI modifications

### PR DESCRIPTION
**Pull request by ottpeter**

### CODE REVIEW

File: src/MainRouter.tsx [hunk 1]
```diff
@@ -42,21 +42,22 @@ const MainRouter = (): ReactElement => {
...
-    try {
-      await fetch(
-        process.env.BEE_API_URL + "bytes/" + process.env.HEALTH_CHECK_DATA_REF
-      );
-      if (!isBeeRunning) {
-        setBeeRunning(true);
-        console.log("Bee is running");
-      }
-    } catch (error) {
+    fetch(
```
Comment (bosi95) [inline src/MainRouter.tsx:45]: why the await statement is removed ?

File: src/pages/Chat/Chat.tsx [hunk 2]
```diff
@@ -31,7 +32,7 @@ interface ChatProps {
 }
 
 const GATEWAY =
-  "86d2154575a43f3bf9922d9c52f0a63daca1cf352d57ef2b5027e38bc8d8f272";
+  "86d2154575a43f3bf9922d9c52f0a63daca1cf352d57ef2b5027e38bc8d8f272"; // this should be in env
```
Comment (bosi95) [inline src/pages/Chat/Chat.tsx:35]: put gateway to .env, vite.config and github secrets. If not set then a dummy or default value can be used.

File: src/pages/ProfileCreation/ProfileCreation.tsx [hunk 3]
```diff
@@ -60,7 +60,7 @@ const ProfileCreation: React.FC = () => {
   };
 
   const saveUsername = async () => {
-    await fetch(process.env.BACKEND_API_URL + "username", {
+    await fetch(process.env.BACKEND_API_URL + "/username", {
```
Comment (bosi95) [inline src/pages/ProfileCreation/ProfileCreation.tsx:63]: do not change tit here, the pattern here must match the .env.sample example

File: src/pages/ProfileCreation/ProfileCreation.tsx [hunk 4]
```diff
@@ -72,7 +72,7 @@ const ProfileCreation: React.FC = () => {
       setIsEdit(false);
       setMonogram(createMonogram(username));
       const response = await fetch(
-        process.env.BACKEND_API_URL + "username/" + username
+        process.env.BACKEND_API_URL + "/username/" + username
```
Comment (bosi95) [inline src/pages/ProfileCreation/ProfileCreation.tsx:75]: same as above

File: src/pages/SpacesPage/SpacesPage.tsx [hunk 5]
```diff
@@ -10,7 +10,7 @@ import { BatchId } from "@ethersphere/bee-js";
 import { useGlobalState } from "../../GlobalStateContext";
 
 // Chat related variables, later this will be deleted
-const STAMP = "5c2a4f00ff17374a9d1ed8e148dc4ed5c3df12f1f0c10d37299db51a3bce0399" as BatchId;
+const STAMP = "53ec9faa9fe03ac14d33f68901dc88d71aa03c810b8de6a11b31f38c1c09d92a" as BatchId;
```
Comment (bosi95) [inline src/pages/SpacesPage/SpacesPage.tsx:13]: put this to .env just like for gateway. btw: there is already a STAMP env var.

File: src/pages/SpacesPage/SpacesPage.tsx [hunk 6]
```diff
@@ -69,7 +69,7 @@ const SpacesPage: React.FC = () => {
         nickname={username}
         gsocResourceId={TestgetResourceId(selectedChat)}
         session={undefined}
-        topMenuColor={undefined && "#F1F2F4"}
+        topMenuColor={undefined}
         originatorPage={"Spaces"}
         originatorPageUrl={"/spaces"}
```
Comment (bosi95) [inline src/pages/SpacesPage/SpacesPage.tsx:74]: use routes.spaces

**Review by bosi95 — CHANGES_REQUESTED** [review]

File: src/pages/ProfileCreation/ProfileCreation.tsx (hunk 4 above)
Comment (ottpeter) [reply src/pages/ProfileCreation/ProfileCreation.tsx:75]: This is so we don't need ending slash in process.env.BACKEND_API_URL

File: src/pages/ProfileCreation/ProfileCreation.tsx (hunk 3 above)
Comment (ottpeter) [reply src/pages/ProfileCreation/ProfileCreation.tsx:63]: This is so we don't need ending slash in process.env.BACKEND_API_URL

File: src/MainRouter.tsx (hunk 1 above)
Comment (ottpeter) [reply src/MainRouter.tsx:45]: I don't know there were some merge conflict here and I might made a mistake when solving it.

Comment (ottpeter) [reply src/MainRouter.tsx]: I put  back await.

**Review by bosi95 — APPROVED** [review]

**Pull request merged**